code/client/launcher/Main.cpp

```diff
@@ -295,16 +297,26 @@ int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, PWSTR pCmdLine
 
 		wcscat_s(systemPath, dll);
 
-		LoadLibrary(systemPath);
+		if(!LoadLibrary(systemPath))
+		{
+			printf("Could not initialize ");

```

Comment:
Would it make more sense to use `OutputDebugString` here? CitiLaunch by default does not have a console window, this is too early for `trace`, but it's typically easier to attach a Win32 global debug monitor than it is to change the process subsystem value.